web-services/client/src/main/java/datawave/webservice/query/metric/BaseQueryMetric.java

```diff
@@ -905,6 +908,10 @@ public String getColumnVisibility() {
     
     public void setColumnVisibility(String columnVisibility) {
         this.columnVisibility = columnVisibility;
```

Comment:
I we are putting this in the markings, do we still nee the columnVisibility member?

Reply:
Decided that the best solution was to leave the setter and getter for markings but not the settings member.  The columnVisibility will hold the value and will be used to polulate a new Map when getMarkings() is called.  Also added serialization tests for XML, JSON, protobuf.